distribution/docs/src/main/resources/_contents/_configuring/configuring-managed-service-factory-bundles-contents.adoc

```diff
@@ -51,14 +50,6 @@ The following table shows the format to use for each property type supported.
 |name = L"value"
 |diameter = L"10000"
 
```

Comment:
❗️ We should mention that floats and doubles are still supported using the IEEE 754 format and still give examples. See https://github.com/livingobjects/felix-configadmin/blob/master/src/main/java/org/apache/felix/cm/file/ConfigurationHandler.java#L435-L443 and https://docs.oracle.com/javase/7/docs/api/java/lang/Float.html#intBitsToFloat(int).